decidim-core/app/controllers/decidim/notifications_settings_controller.rb

```diff
@@ -10,6 +10,28 @@ def show
       @notifications_settings = form(NotificationsSettingsForm).from_model(current_user)
     end
 
+    def unsubscribe
+      authorize! :update, current_user
+      @unsubscribe_notifications_settings = false
```

Comment:
Why do you need this? If the `UnsubscribeNotificationsSettings` command is only used to unsubscribe, then this param is redundant.